core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -708,6 +721,7 @@ case class QualificationSummaryInfo(
     allClusterTagsMap: Map[String, String],
     mlFunctions: Option[Seq[MLFunctions]],
     mlFunctionsStageDurations: Option[Seq[MLFuncsStageDuration]],
+    unsupportedExecstoExprsMap: Map[String, String],
```

Comment:
As we added `unsupportedExecExprsMap` to class `QualificationSummaryInfo`; should we add also the field into `TestQualificationSummary` so that mock implementations in unit tests is up-to-date with the actual execution?
I remember that at some point we found out that the unit tests were not accurate because `TestQualificationSummary` was ignoring many columns.

Reply:
I don't think we need to add unsupportedExecExprsMap to  TestQualificationSummary. That is used for testing output for ApplicationSummary. We do not include unsupportedExecExprsMap  in the output file. So leaving it should be fine. Please let me know if you think otherwise.